reports/report_builder.py

```diff
@@ -4106,10 +4106,46 @@ def ws_s27(self, ws):
             c = ws.dim_colmax + 2
     
     def ws_s28(self, ws):
-        return
-    
+        """
+        Cols: Sex of subject
+        Rows: Country
+        """
+        counts = Counter()
+        for _, model in person_models.items():
+            sheet_name = model.sheet_name()
+            country_field = f"{sheet_name}__country"
+            rows = model.objects \
+                    .values('sex', country_field) \
+                    .filter(**{f"{sheet_name}__covid19": 1}) \
+                    .filter(**{f"{country_field}__in": self.country_list}) \
+                    .filter(sex__in=self.male_female_ids) \
+                    .annotate(n=Count('id'))
+
```

Comment:
we don't have weights for these sheets, right?

Reply:
Yeah, I've looked at all of them (`s` and `sr`) and none has weights applied @esirK . Please double check and let me know if you spot any.

... This is something we'll need to confirm in this week's meeting (along with Regional weights).

Reply:
I also can't find any.